scripts/controls.js

```diff
@@ -138,28 +139,28 @@ export function handleMouseMove(event) {
 }
 
 export function handleKeyPress(event) {
-    if (!allowInput) {
-        return;
-    }
-    const keyCode = event.keyCode;
+    if (allowInput && !isGameOver) {
+        const keyCode = event.keyCode;
 
-    // Left arrow key
-    if (keyCode === 37) {
-        moveFish("left");
-    }
-    // Right arrow key
-    else if (keyCode === 39) {
-        moveFish("right");
-    }
-    // Down arrow key
-    else if (keyCode === 40 || keyCode == 32) {
-        dropFish();
+        // Left arrow key
+        if (keyCode === 37) {
```

Comment:
what the logic behind this

Reply:
the gameOver boolean makes it so that the fish cant move around or anything when game over screen shows up